app/importers/envirotech/regulation_data.rb

```diff
@@ -15,8 +15,9 @@ class RegulationData < Envirotech::BaseData
       'url'             => :url,
     }.freeze
 
-    def initialize(resource = ENDPOINT)
+    def initialize(resource = ENDPOINT, relation_data: nil)
```

Comment:
I don't know why I don't use named arguments all the time :-) I think I'll start doing so.